sample-functions/HostnameIntent/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM alpine:latest
```

Comment:
We need to change these references to 3.6, that's why they failed to build since latest was pointing at 3.5 and is now 3.6. Can take it in a follow-up PR though.